backend/infrahub/core/schema/definitions/internal.py

```diff
@@ -269,7 +270,7 @@ def to_dict(self) -> dict[str, Any]:
         SchemaAttribute(
             name="default_filter",
             kind="Text",
-            regex=str(NAME_REGEX),
+            regex=str(NAME_REGEX_OR_EMPTY),
             description="Default filter used to search for a node in addition to its ID. (deprecated: please use human_friendly_id instead)",
```

Comment:
It would be good to add documentation to enlight that specifying an empty string allows to remove an existing value but it currently breaks the 128 characters max length of this attribute https://github.com/opsmill/infrahub/actions/runs/14246718903/job/39929589165?pr=6226